lib/travis/build/script/d.rb

```diff
@@ -18,22 +19,34 @@ def setup
           super
 
           sh.echo 'D support for Travis-CI is community maintained.', ansi: :green
-          sh.echo 'Please make sure to ping @MartinNowak, @klickverbot and @ibuclaw '\
+          sh.echo 'Please make sure to ping @MartinNowak and @wilzbach'\
             'when filing issues under https://github.com/travis-ci/travis-ci/issues.', ansi: :green
 
+          sh.echo 'DMD-related issues: https://issues.dlang.org', ansi: :green
+          sh.echo 'LDC-related issues: https://github.com/ldc-developers/ldc/issues', ansi: :green
+          sh.echo 'GDC-related issues: https://bugzilla.gdcproject.org', ansi: :green
+          sh.echo 'DUB-related issues: https://github.com/dlang/dub/issues', ansi: :green
+
           sh.fold 'compiler-download' do
             sh.echo 'Installing compiler and dub', ansi: :yellow
 
             sh.cmd 'CURL_USER_AGENT="Travis-CI $(curl --version | head -n 1)"'
             sh.cmd <<-'EOT'.gsub(/^              /, '')
+              __mirrors=(
+                "https://dlang.org/install.sh"
+                "https://downloads.dlang.org/other/install.sh"
+                "https://nightlies.dlang.org/install.sh"
+                "https://github.com/dlang/installer/raw/stable/script/install.sh"
+              )
               for i in {0..4}; do
-                  if curl -fsS -A "$CURL_USER_AGENT" --max-time 5 https://dlang.org/install.sh -O ||
-                          curl -fsS -A "$CURL_USER_AGENT" --max-time 5 https://nightlies.dlang.org/install.sh -O; then
-                      break
-                  elif [ $i -lt 4 ]; then
-                      sleep $((1 << $i))
-                  fi
+                for mirror in "${__mirrors[@]}" ; do
+                    if curl -fsSL -A "$CURL_USER_AGENT" --connect-timeout 5 --speed-time 30 --speed-limit 1024 "$mirror" -O ; then
```

Comment:
Indentation seems inconsistent here.

Reply:
I saw you already merged this PR. Can I still fix this? (i.e. in a quick follow-up PR?)